### PR TITLE
Colors migration - 1 - Reorganization

### PR DESCRIPTION
**Pull request by damianmef**

### CODE REVIEW

File: lib/js/components/Cards/CardExpandable/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './CardExpandable.spec';
```
Comment (kvas-damian) [inline lib/js/components/Cards/CardExpandable/index.ts:1]: do we need to export tests?

File: lib/js/index.ts [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-export { default as Card } from './components/Cards/Card.vue';
+export { default as Badge } from './components/Badges/Badge';
+export * from './components/Badges/Badge';
+export * from './components/Badges/Badge/Badge.consts';
+export { default as BadgeScore } from './components/Badges/BadgeScore';
+export * from './components/Badges/BadgeScore/BadgeScore.consts';
+export { default as Button } from './components/Buttons/Button';
+export * from './components/Buttons/Button/Button.consts';
+export { default as Card } from './components/Cards/Card/Card.vue';
 export { default as CardExpandable } from './components/Cards/CardExpandable.vue';
```
Comment (kvas-damian) [inline lib/js/index.ts:9]: invalid path

File: lib/js/components/Cards/CardExpandable/index.ts (hunk 1 above)
Comment (damianmef) [reply lib/js/components/Cards/CardExpandable/index.ts:1]: To check by me.

Comment (damianmef) [reply lib/js/components/Cards/CardExpandable/index.ts:1]: I have no explanation for this, I have found no reason for this to stay in the project. In the corresponding `Card` folder there is no such thing. So It's going out.

File: lib/js/index.ts (hunk 2 above)
Comment (damianmef) [reply lib/js/index.ts]: Fixed.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**